SECURITY.md

```diff
@@ -0,0 +1,8 @@
+## Reporting security issues
+
+The OME team and community take security bugs seriously.
+
+If you discover a security vulnerability or would like to report a security issue privately and securely, please email us at security@openmicroscopy.org. You can use GPG keys to communicate with us securely.
```

Comment:
Can you link to the key?

Reply:
Makes sense. We don't have a direct anchor to the website though? Or did you mean linking to a registry like https://pgp.key-server.io/pks/lookup?search=security%40openmicroscopy.org&fingerprint=on&op=vindex ?

Reply:
Anything that enables people to get the key directly, or to verify the fingerprint. AFAIK anyone can create a key with any email address so a searching for an email is insufficient.

Reply:
Definitely can't trust the email address alone. I'd think a link to https://www.openmicroscopy.org/security/ (or potentially with an added anchor of `#gpg-key`) would suffice.

Reply:
See ca235e2